src/Admins.vue

```diff
@@ -58,7 +65,9 @@ export default {
 	},
 	methods: {
 		openNewAdmin() {
-			this.isAdding = true
+			setTimeout(() => {
```

Comment:
could you elaborate and add a comment why this is necessary? Did you want to use https://vuejs.org/v2/api/#vm-nextTick?

Reply:
it works, so let's merge this now. But @georgehrke please clarify afterwards :)

Reply:
>  Did you want to use https://vuejs.org/v2/api/#vm-nextTick?

No, because nextTick is only executed after changing data, once the next DOM rendering is completed. I'm not manipulating the data here.

I'm using the timeout to make sure, the click is not propagated anymore, before changing the data.